package.json

```diff
@@ -2,7 +2,8 @@
   "name": "root",
   "private": true,
   "workspaces": [
-    "packages/*"
+    "packages/*",
+    "docs"
```

Comment:
we add the docs as a separate workspace in the monorepo. Not entirely sure why but create-react-app has it setup this way https://github.com/facebook/create-react-app/blob/main/package.json#L3